internals/core_models.py

```diff
@@ -314,6 +314,25 @@ def format_for_template(self, version=2):
       else:
         d['browsers']['chrome']['status']['milestone_str'] = d['browsers']['chrome']['status']['text']
 
+      # The followings are for the guide forms
+      d['format_for_edit'] = {
```

Comment:
Can this be done in JS using the existing fields?
Usually an API should be aligned with the data models rather than being biased toward any one particular UX.   Adding a lot of editing-specific fields will increase the transfer time for other use cases.

Since logic to format these fields for editing would likely be used in several elements, it could be located in utils.js or one of the forms-related JS files.

Reply:
OK. That makes sense. I will work on this!

Reply:
I've made the requested changes. Please review it again.

I "un-popped" a lot of fields in format_for_template(), this makes formatting for edit on the frontend easier but adds redundant fields.

The other approach is to still pop these fields but in JS, do more mappings between feature objects for view and for edit. For example:
```js
formattedFeature['bug_url'] = feature['browsers']['chrome']['bug']
formattedFeature['blink_components'] = feature['browsers']['chrome']['blink_components']
formattedFeature['devrel'] = feature['browsers']['chrome']['devrel']
formattedFeature['owner'] = feature['browsers']['chrome']['owners']
```

This would affect many following PRs as well. Please let me know what you think!